api/tests/Feature/CommunityInterestTest.php

```diff
@@ -430,24 +409,28 @@ public function testCommunityInterestFilter(): void
         $communityInterestWithJobInterest = CommunityInterest::factory()->create([
             'user_id' => User::factory(),
             'community_id' => $this->communityId,
+            'consent_to_share_profile' => true,
             'job_interest' => true,
             'training_interest' => false,
         ]);
         $communityInterestWithTrainingInterest = CommunityInterest::factory()->create([
             'user_id' => User::factory(),
             'community_id' => $this->communityId,
+            'consent_to_share_profile' => true,
             'job_interest' => false,
             'training_interest' => true,
         ]);
         $communityInterestWithBothInterests = CommunityInterest::factory()->create([
             'user_id' => User::factory(),
             'community_id' => $this->communityId,
+            'consent_to_share_profile' => true,
             'job_interest' => true,
             'training_interest' => true,
         ]);
         $communityInterestWithNoInterests = CommunityInterest::factory()->create([
             'user_id' => User::factory(),
             'community_id' => $this->communityId,
+            'consent_to_share_profile' => false,
```

Comment:
I think consent_to_share_profile should be true here since otherwise the communityInterest shouldn't show up at all.

Reply:
Oh, good catch.  That surfaced a bug.
[fix tests](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13146/commits/3e86c896839af4a785762183171b2a2b37852925)